### PR TITLE
Polishing and typos

### PR DESCRIPTION
**Pull request by ashamukov**
Hello! A few small fixes here.
Notes:
* It can also be merged into 0.7-maintenance (to fix typos on the website)
* tf-idf-full.png has difference: doc-lines -> doc-count (in the leftmost block)

By the way, do you have sources for png block schemas?

### CODE REVIEW

File: src/docs/asciidoc/expert-zone/custom-source-sink.adoc [hunk 1]
```diff
@@ -50,7 +50,7 @@ of.
 
 The client serializes an instance of `ProcessorMetaSupplier` as part of
 each `Vertex` in the `DAG`. The **coordinator** member deserializes the
-instance and uses it to create `ProcessorSupplier`s by calling the
+instance and uses it to create `ProcessorSupplier` 's by calling the
```
Comment (viliam-durina) [inline src/docs/asciidoc/expert-zone/custom-source-sink.adoc]: This isn't possessive, but plural, should be without the `'`.

File: src/docs/asciidoc/pipeline-api/pipeline-api.adoc [hunk 2]
```diff
@@ -422,8 +422,8 @@ apply an aggregate function to all the grouped items, separated by the
 originating stream. The aggregate function can produce a summary value
 like an average or linear trend, but it can also keep a list of all the
 items, effectively <<join-without-aggregating, avoiding actual
-aggregation>>. You can achieve any kind of join: inner/outer/full, left
-and right. The constraint, as already noted, is that the join condition
+aggregation>>. You can achieve any kind of join: inner/outer(full, left
+and right). The constraint, as already noted, is that the join condition
```
Comment (viliam-durina) [inline src/docs/asciidoc/pipeline-api/pipeline-api.adoc]: There should be space before `(`.

Comment (ashamukov) [reply src/docs/asciidoc/pipeline-api/pipeline-api.adoc]: Fixed, thanks

**Comment by cangencer** [comment]
verify

**Comment by cangencer** [comment]
@ashamukov can you rebase/update the branch with master so that the build passes?

**Comment by cangencer** [comment]
verify

**Comment by viliam-durina** [comment]
Thanks for the PR 👍

**Pull request merged**